src/fastoad/notebooks/02_explanation/02_pure_python.ipynb

```diff
@@ -0,0 +1,836 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.    print("The aspect ratio which gives the minimum MTOW is ", np.round(min_MTOW_AR[0], 2))
</pre><p>At least 3 significant digits (or 4 to better spot discrepancies with next notebooks)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
The number of significant number have been changed for MDO so that the difference with the optimization by hand is clearer

Reply:
Changed in dfe233f7